test/test_gradio_component_cli.py

```diff
@@ -4,39 +4,31 @@
 
 import pytest
 
-from gradio.cli.commands.components._create_utils import OVERRIDES
 from gradio.cli.commands.components.build import _build
-from gradio.cli.commands.components.create import _create
+from gradio.cli.commands.components.create import _create, _create_utils
 from gradio.cli.commands.components.install_component import _get_executable_path
 from gradio.cli.commands.components.publish import _get_version_from_file
 from gradio.cli.commands.components.show import _show
+from gradio.utils import core_gradio_components
+
+core = [
+    c.__name__
+    for c in core_gradio_components()
+    if not getattr(c, "is_template", False)
+    and c.__name__ not in ["Tab", "Form", "FormComponent"]
+]
 
 
 @pytest.mark.parametrize(
     "template",
-    [
-        "Row",
-        "Column",
-        "Tabs",
-        "Group",
-        "Accordion",
-        "AnnotatedImage",
-        "HighlightedText",
-        "BarPlot",
-        "ClearButton",
-        "ColorPicker",
-        "DuplicateButton",
-        "LinePlot",
-        "LogoutButton",
-        "LoginButton",
-        "ScatterPlot",
-        "UploadButton",
-        "JSON",
-        "FileExplorer",
-        "Model3D",
-    ],
+    core,
 )
 def test_template_override_component(template, tmp_path):
+    """When you add a new component this test will likely fail locally
+    because the js files have not been moved to the _frontend_code directory.
+
+    Just build the python package to move the latest state of the js directory to _frontend_code.
```

Comment:
What does "build the python package" mean? Do you mean build the frontend?

Reply:
Need to build the wheel so that a script to copy the js files gets run. Let me update comment!